app/src/main/java/com/randomappsinc/studentpicker/home/HomeActivity.java

```diff
@@ -133,6 +141,17 @@ private void showPleaseRateDialog() {
                 .show();
     }
 
+    private void showBackupAndDialogFeature() {
```

Comment:
nit: We're just showing the dialog, not the feature itself.
```suggestion
    private void showBackupAndRestoreDialog() {
```